### PR TITLE
Validate KEANU

### PR DESCRIPTION
**Pull request by KeanuCoin**
Attestations:
*Validation on X: https://x.com/Keanu_Bull/status/1776637025350750296 
*Coingecko/CMC URL: Application has been made to main sites. Dex scanners: https://www.geckoterminal.com/solana/pools/4AtL9cjyaMFnU7qopKaMWJJGQAmu5L3QZN9FBfX6aDko, https://coinmarketcap.com/dexscan/solana/4AtL9cjyaMFnU7qopKaMWJJGQAmu5L3QZN9FBfX6aDko/
*Token Icon: https://ibb.co/gTBxxz4
*Token Decimals: 6

Validation:
[x] The metadata provided in the PR matches what is on-chain 
[x] Does not duplicate the symbol of another token on Jupiter's strict list 
[ ] Is Listed on Coingecko / CMC

Acknowledgement:
[x] I did not modify unrelated code.
[x] My change matches the format in the file (no spaces between fields). 
[x] My token is already live and trading on Jupiter. 
[x] I read the README section on Community-Driven Validation and will get community support on Twitter.

### CODE REVIEW

**Comment by KeanuCoin** [comment]
Original Wif Founder Patrick Bateman has received an allocation and is supporting the project. Original Wif dev Arashii has joined the team and is now our main coder. See https://keanu.bio and https://t.me/keanu_coin.

File: validated-tokens.csv
Comment (joshitachu) [inline validated-tokens.csv:1]: Keanu is a solid project with a long term focus !

**Review by joshitachu — COMMENTED** [review]
Solid project with a long term focus

File: validated-tokens.csv
Comment (SolanaFan) [reply validated-tokens.csv:1]: Excellent project with Wif Founder and Dev on board.

**Review by SolanaFan — COMMENTED** [review]
$KEANU is bullish with great community support and over 25k following on socials.

**Review by SolanaFan — COMMENTED** [review]
Good

**Comment by Jymfletzfur** [comment]
duplicate submission, there is another one 2631

**Pull request closed without merging**